dateformatterutil/src/main/java/com/example/dateformatter/DateFormatterUtil.kt

```diff
@@ -0,0 +1,182 @@
+        val format = SimpleDateFormat("HH:mm")
+        format.timeZone = TimeZone.getTimeZone("UTC")
+        val sdf = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss")
+        var d: Date? = null
+        try {
+            d = sdf.parse(dateString)
+        } catch (e: ParseException) {
+            e.printStackTrace()
+        }
+        return format.format(d)
+    }
+    /**
+     * @usage this function is used to convert date string to days and months
+     * @param date A String containing the date
+     * @return A String that represents the days and months
+     */
+    fun convertDateToDateDaysMonthsUTC(date: String): String {
+        val format = SimpleDateFormat("dd MMMM")
+        format.timeZone = TimeZone.getTimeZone("UTC")
+        val sdf = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss")
+        var d: Date? = null
+        try {
+            d = sdf.parse(date)
+        } catch (e: ParseException) {
+            e.printStackTrace()
+        }
+        return format.format(d)
+    }
+    /**
+     * @usage this function is used to compare two date
+     * @param fromDate A String containing the start date
+     * @param toDate A String containing the end date
+     * @return A Boolean that indicates the end date is after start or not
+     */
+    fun compareDates(fromDate: String , toDate: String): Boolean {
+        return Date(toDate).after(Date(fromDate))
+    }
+    /**
+     * @usage this function is used to split hours and min from date based on am and pm
+     * @param date A String containing the date
+     * @return A String that contains hh:min am/pm
+     */
+    fun getDatFromPmAmHoursMin(date: String): String {
+        val h1 = date.split(":".toRegex()).toTypedArray()
+        val hour = h1[0]
+        val minute = h1[1]
+        return if (date.contains("PM")) "$hour:$minute PM" else "$hour:$minute AM"
+    }
+    /**
+     * @usage this function is used to get current date
+     * @return A String that represents the current date
+     */
+    fun getCurrentData(): String {
+        val c = Calendar.getInstance().time
+        val df = SimpleDateFormat("yyyy/MM/dd", Locale.ENGLISH)
+        return df.format(c)
+    }
+    /**
+     * @usage this function is used to convert time from mill sec to sec
+     * @param georgianDate A String containing the date
+     * @return A String that represents the sec of given date
+     */
+    fun convertTimeFromMillSecondToSecond(georgianDate: String): String{
+        try {
+            val calendar = Calendar.getInstance()
+            val df: java.text.DateFormat =
+                SimpleDateFormat("yyyy/MM/dd - hh:mm:ss a", Locale.ENGLISH)
+            val dateLong = df.parse(georgianDate).time
+            val formatter = SimpleDateFormat("hh:mm a", Locale.ENGLISH)
+            calendar.timeInMillis = dateLong
+            return formatter.format(calendar.time)
+        } catch (e: ParseException) {
+            e.printStackTrace()
+        }
+        return ""
+    }
```

Comment:
First: Use an appropriate function naming as the current function name doesn't represent the actual usage of the function.
Second: The user can't figure out he must pass a dateString with specific format that the function accepts "yyyy/MM/dd - hh:mm:ss a"
=> for example: what should happen if he passes a dateString with format of  "yyyy-MM-dd'T'HH:mm:ss" ?